graphserver/server.go

```diff
@@ -30,6 +31,26 @@ func NewArachneBadgerServer(baseDir string) *ArachneServer {
 	}
 }
 
+func NewArachneBoltServer(baseDir string) *ArachneServer {
+	a, err := kvgraph.NewKVArachne("bolt", baseDir)
```

Comment:
Are you sure this string-based wrapper is necessary? Especially since you're wrapping it back in a type-specific constructor?

Reply:
It's a soft link. If the code is built with `--tags rocks` then the rocks.go code is built, and the driver is added to the map of avalible drivers at https://github.com/bmeg/arachne/blob/kvdriver/rocksdb/rocks.go#L41